CHANGELOG.md

```diff
@@ -2,7 +2,13 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
-## 6.2.12
+## 6.2.13
+#### Fixed
+- Make `IterablePushNotificationMetadata` struct public.
```

Comment:
Most release notes below used past tense for the changes (or present but to describe the current state). I'd change this to keep it more consistent.